CONTRIBUTING.md

```diff
@@ -1,11 +1,51 @@
+
+## Want a new feature?
+
+> For raising a feature request please use the template provided as a guideline, provided to you.
+
+## Branch Organization
+
+Submit all changes directly to the `master` branch. We don’t use separate branches for development or for upcoming releases. We do our best to keep master in good shape, with all tests passing.
+
+Code that lands in master must be compatible with the latest stable release. It may contain additional features, but no breaking changes. We should be able to release a new minor version from the tip of master at any time.
+
+## Code structure
+
+Currently the project is managed with lerna and it contains 2 packages:
+
+- [graphql-kimera](https://github.com/lola-tech/graphql-kimera/tree/master/packages/graphql-kimera): containing the source code and tests for the module published on npm.
+- [graphql-kimera-docs](https://github.com/lola-tech/graphql-kimera/tree/master/packages/graphql-kimera-docs): a [docusaurus](https://v2.docusaurus.io/) containing the docusaurus app for managing and changing the documentation.
+
+> Most of the development work will happen on [graphql-kimera](https://github.com/lola-tech/graphql-kimera/tree/master/packages/graphql-kimera) package.
+
+## Your first pull request
+
+> **Working on your first Pull Request?** You can learn how from this _free_ series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it so other people don’t accidentally duplicate your effort.
+
+If somebody claims an issue but doesn’t follow up for more than two weeks, it’s fine to take it over but you should still leave a comment.
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device.
+2. Create a new branch `git checkout -b MY_BRANCH_NAME`
+3. Install the dependencies: `npm install` and you should be good to go.
+4. Navigate to the module folder from the project root `cd packages/graphql-kimera/`.
+
+## Sending a Pull Request
 
-Kimera team has adopted the Contributor Covenant as its Code of Conduct, and we expect project participants to adhere to it. Please read the full text so that you can understand what actions will and will not be tolerated.
+We will review your pull request and either merge it, request changes to it, or close it with an explanation. We’ll do our best to provide updates and feedback throughout the process.
 
-## Open Development
+Before submitting a pull request, if you’ve fixed a bug or added code that should be tested, add tests! Ensure the test suite passes (`npm test`).
```

Comment:
I'd go with : "Test your code before submitting a pull request! If you’ve fixed a bug or added code for a new feature, ensure the test suite passes (`npm test`)."